app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    pass

```

Comment:
The `NotWearingMaskError` class should include a more descriptive error message. Consider overriding the `__init__` method to provide additional context about the error.